SND/EmulsionTarget/Target.cxx

```diff
@@ -528,7 +528,7 @@ Bool_t Target::ProcessHits(FairVolume* vol)
                pdgCode);
 
         // Increment number of muon det points in TParticle
-        ShipStack* stack = (ShipStack*)gMC->GetStack();
+        ShipStack* stack = static_cast<ShipStack*>(gMC->GetStack());
```

Comment:
this could be `auto stack = static_cast<ShipStack*>(gMC->GetStack());` as the type is explicit in the `static_cast<>` for consistency.